docs/specification.md

```diff
@@ -54,6 +54,11 @@ The `nodes\props` group is optional and will contain one or more `node property`
 
     When writing a graph with missing properties to the geff format, you must fill in a dummy value in the `values` array for the nodes that are missing the property, in order to keep the indices aligned with the node ids. 
 
+### The `vlen_props` group and `node vlen property` groups
+The `nodes\vlen_props` group is optional and will contain one or more `node vlen property` groups, each with a `values` array, a `slices` array and an optional `missing` array. These properties are stored as a list of arrays, where each element in the array can have a variable length. This is useful for properties that are not fixed-length, such as polygons or other complex shapes.
```

Comment:
Ideally, the vlen_props would be as similar to normal properties as possible. I would encourage them to be within the same group. I would even rename the `slices` array to be the `values` so that we can re-use our validation functions. The proposed `values` can be something like `data`.